iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/util/ModificationReports.java

```diff
@@ -645,4 +645,31 @@ public static void scalingReport(Reporter reporter, String type, ScalingType sca
             .withSeverity(TypedValue.INFO_SEVERITY)
             .build());
     }
+
+    public static void connectableConnectionReport(Reporter reporter, Connectable<?> connectable, boolean connectionSuccessful) {
+        String defaultMessage = connectionSuccessful ?
+            "Connectable ${connectable} has been connected." :
+            "Connectable ${connectable} has NOT been connected.";
+        String key = connectionSuccessful ? "connectableConnected" : "connectableNotConnected";
+        reporter.report(Report.builder()
+            .withKey(key)
+            .withDefaultMessage(defaultMessage)
+            .withValue("connectable", connectable.getId())
+            .withSeverity(TypedValue.INFO_SEVERITY)
+            .build());
+    }
+
+    public static void connectableDisconnectionReport(Reporter reporter, Connectable<?> connectable, boolean disconnectionSuccessful, boolean isPlanned) {
+        String defaultMessage = disconnectionSuccessful ?
+            "Connectable ${connectable} has been disconnected." :
+            "Connectable ${connectable} has NOT been disconnected.";
+        String key = isPlanned ? "planned" : "unplanned";
```

Comment:
isPlanned doesn't change the message so either add it to the message or remove it from the key